core/src/services/monoiofs/backend.rs

```diff
@@ -31,8 +36,16 @@ pub struct MonoiofsConfig {
     ///
     /// All operations will happen under this root.
     ///
-    /// Default to `/` if not set.
+    /// Builder::build will return error if not set.
     pub root: Option<String>,
+    /// Count of worker threads that each runs a monoio runtime.
+    ///
+    /// Default to 1.
+    pub worker_threads: Option<NonZeroUsize>,
```

Comment:
It's a bit odd for OpenDAL to expose such low-level configurations. Let's set a hardcoded number for these and remove those configurations. We can add them back if there is user demand.

Reply:
These low-level configurations are removed in the latest commit. Please review it again. Thanks.